pkg/log/log_context.go

```diff
@@ -0,0 +1,67 @@
+package log
+
+import (
+	"context"
+)
+
+var allowList map[string]bool
+
+func AllowContextFields(fields ...string) {
```

Comment:
This should have a comment in the form of `// AllowContextFields ...` as it's exported